pkg/network/node/ovscontroller_test.go

```diff
@@ -968,6 +991,79 @@ func TestFindUnusedVNIDs(t *testing.T) {
 	}
 }
 
+func TestFindPolicyVNIDs(t *testing.T) {
+	testcases := []struct {
+		flows       []string
+		policyVNIDs sets.Int
+	}{
+		{
```

Comment:
This test is wrong:

https://github.com/openshift/sdn/pull/103/commits/f5c2488a9475cca70e2a3bad76add6915ef54726#diff-8f7a8126747765bebab51b916de23c99R1000-R1005

Reply:
Looks like I'm highlighting the wrong line. I meant this test:
`
			// 0 must always be present
			flows: []string{
				"table=80,priority=0 actions=drop",
			},
			policyVNIDs: sets.NewInt(0x0),
		},`